Sources/NIO/Bootstrap.swift

```diff
@@ -470,18 +517,28 @@ public final class DatagramBootstrap {
         return self
     }
 
+    /// Use the existing bound socket file descriptor.
+    ///
+    /// - parameters:
+    ///     - descriptor: The _Unix file descriptor_ representing the bound datagram socket.
+    public func withBoundSocket(descriptor: CInt) -> EventLoopFuture<Channel> {
+        return bind0(makeChannel: { eventLoop in
+            return try DatagramChannel(eventLoop: eventLoop, descriptor: descriptor)
+        }) { (eventLoop, channel) in
```

Comment:
Same minor note here, might be nicer to pass this first closure by name as an inner function.

Reply:
Same thing, it captures the descriptor.